src/UIAutomation/UiaOperationAbstraction/UiaTypeAbstractionImpl.g.cpp

```diff
@@ -7008,7 +7008,10 @@
             auto element = std::get<AutomationRemoteElement>(m_member).GetParentElement();
             if (cacheRequest)
             {
-                element.PopulateCache(*cacheRequest);
+                delegator->If(element.IsNull().BoolNot(), [&]()

```

Comment:
Could we add tests to validate this scenario?

Reply:
yes, I will add a test to validate this.